python/avogadro/cjson.py

```diff
@@ -0,0 +1,41 @@
+"""
+/******************************************************************************
+  This source file is part of the Avogadro project.
+  This source code is released under the New BSD License, (the "License").
+******************************************************************************/
+"""
+import json
+class Cjson:
+    """
+    This Class is intended to read Cjson files
+    with python libraries and perform certain
+    methods on files and convert them back to Cjson
+    files as required
+    """
+    def __init__(self):
+        pass
+    def __open_file(self, filePath):
```

Comment:
Generally when we're creating modules, we want symmetry in names. So if you're creating an `__open_file()` method, there should be a save version. So if you want `__to_cjson` then you want `__from_cjson`.

Or what if a user has a string (e.g. from a URL or from another source)?

So you might want to check if `filePath` can be considered a file path and if not, try loading it as a json string.

Reply:
Sure! I will keep this in mind from now on.

I will check for strings and links as you told 
currently It was for the filepath of file present in the system